docs/src/05-install/install.md

```diff
@@ -49,4 +87,43 @@ Then run the following command to build the container image.
 ./cp-deploy.sh build
 ```
 
-This process will take 5-10 minutes to complete and it will install all the pre-requisites needed to run the automation, including Ansible, Python and required operating system packages. For the installation to work, the system on which the image is built must be connected to the internet.
\ No newline at end of file
+This process will take 5-10 minutes to complete and it will install all the pre-requisites needed to run the automation, including Ansible, Python and required operating system packages. For the installation to work, the system on which the image is built must be connected to the internet.
+
+# Getting Cloud Pak Deployer Image from Quay.io
+You can obtain the Cloud Pak Deployer image from the Quay.io container registry. This method allows you to fetch the image directly from the repository. Here's a step-by-step guide on how to do this:
+
+1. Login to Quay.io
+Before pulling the Cloud Pak Deployer image, you need to log in to your Quay.io account or create one if you don't have it.
+```
+docker login quay.io
+```
+
+2. Pull the Cloud Pak Deployer Image
+
+Using Docker : 
+```
+docker pull quay.io/cloud-pak-deployer/cloud-pak-deployer
+```
+
+Using podman 
+```
+podman pull quay.io/cloud-pak-deployer/cloud-pak-deployer
+```
+
+# Air-gapped :
```

Comment:
Air-gapped deployment covered in the "advanced" section. Deployer copies its image to the status directory.